chain/client/src/client.rs

```diff
@@ -1609,7 +1609,17 @@ impl Client {
             BlockStatus::Next => {
                 // If this block immediately follows the current tip, remove
                 // transactions from the tx pool.
-                self.remove_transactions_for_block(validator_id, block).unwrap_or_default();
+                match self.remove_transactions_for_block(&validator_id, block) {
+                    Ok(()) => (),
+                    Err(err) => {
+                        tracing::debug!(
```

Comment:
nit: can you add `target: "client"` in the new logs?

Reply:
fixed.